### ISSUE TITLE
gremlin-server.sh produces debug output by default

### ISSUE THREAD

**Issue by FlorianHockmann** [JanusGraph/janusgraph#1189]
The `gremlin-server.sh` script distributed with JanusGraph [contains this line](https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-dist/src/assembly/static/bin/gremlin-server.sh#L57):
```sh
set -x
```
which means that the command used to start the server with all its arguments will be printed. This leads to unnecessary debug output. The [corresponding file in TinkerPop's repo](https://github.com/apache/tinkerpop/blob/master/gremlin-server/src/main/bin/gremlin-server.sh#L32) does this only when a debug flag is set:
```sh
[[ -n "$DEBUG" ]] && set -x
```

In environment where everything printed to console is collected as logs like frequently done with Docker, This leads to uinteresting and hard to parse log entries as the format can't be configured.

I suggest that this line is simply removed from the script or only activated when a debug flag is set like how it's done in TinkerPop.

**Comment by DivyanshuSaxena** [comment]
Is there any update on this issue?
I am using the JanusGraph Maven dependency and this line cannot be changed there.
